get-started/index.md

```diff
@@ -62,7 +65,302 @@ the specifics of how to deploy your Cryostat instance. Continue to [Setup](#setu
 Note: Alternative methods for installing the operator are described in [Alternate Installation Options](/alternate-installation-options) (not recommended).
 ## [Setup](#setup)
 
-### Deploying Cryostat
+### [Configuring Applications](#configuring-applications)
+The following sections will briefly describe how to configure your Java applications so that Cryostat is able to discover and monitor them.
+These examples will assume the application is built with Maven, packaged into an image with a `Dockerfile`, and running in Kubernetes,
+but the instructions will be similar for other toolchains and platforms as well.
+
+#### [Using the Cryostat Agent](#using-the-cryostat-agent)
+
+[The Cryostat Agent](/guides/#using-the-cryostat-agent)
+is compatible with Cryostat versions 2.3.0 and newer, and application JDKs 11 and newer. If you are using an older version of Cryostat, please upgrade.
+If your application uses a recent version of JDK8 with JFR support, please either upgrade to JDK11+ or [continue to the next section](#using-jmx)
+to learn how to configure your application without the Cryostat Agent.
+
+The Cryostat Agent JAR must be available to your application JVM. The JAR asset can be downloaded [directly from upstream](https://github.com/cryostatio/cryostat-agent/releases),
+or you may use the following snippet in your `pom.xml` to streamline this.
+
+```xml
+<project>
+  ...
+  <repositories>
+    <repository>
+      <id>github</id>
+      <url>https://maven.pkg.github.com/cryostatio/cryostat-agent</url>
+    </repository>
+  </repositories>
+  ...
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>3.3.0</version>
+        <executions>
+          <execution>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>copy</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>io.cryostat</groupId>
+                  <artifactId>cryostat-agent</artifactId>
+                  <version>0.2.0</version>
+                </artifactItem>
+              </artifactItems>
+              <stripVersion>true</stripVersion>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+    ...
+  </build>
+  ...
+</project>
+```
+
+**Note**: You may be required to [authenticate to the GitHub Maven Packages registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry#installing-a-package) in order to pull this JAR.
```

Comment:
Should it be this link instead?

https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry#authenticating-to-github-packages

Reply:
My link has a link that goes to the one you've linked :-) I think either would be fine, but the one I linked gives a little more context first reinforcing what I had written, before going over to the specific new details that aren't written on our site.

Reply:
oh yeh that makes more sense! Thanks!